### PR TITLE
Update ergocubSN000 battery conf files

### PR DESCRIPTION
**Pull request by MSECode**
This PR brings the following changes for ergoCubSN000 (I suppose these can be copied as they are in ergocubSN001 - after revision):
- Add conf files for bat and bms in hw and wrapper
- Update the yarprobotinterface conf files
- Set to use CAN2:1
- Define separate ports on wrapppers for bat and bms devices

### CODE REVIEW

**Comment by MSECode** [comment]
cc: @valegagge 
Correlated to one of the points of issue: https://github.com/icub-tech-iit/project-ergocub/issues/129

**Review by pattacini — COMMENTED** [review]
Ok with me 👍🏻 
I'll wait for @valegagge's review before merging.

After her review, @MSECode, can you please update the PR in order to already ship the same updates for ergoCubSN001?

**Review by pattacini — CHANGES_REQUESTED** [review]
Hi @MSECode 

The CI failed with the following outputs:

### XML consistency check
```console
1 - test yarprobotinterface.ini presence **************

yarprobotinterface.ini FOUND! PASSED

2 - test included xml files presence **************

found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/POS/right_hand-pos_wrapper4.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/POS/right_hand-pos4.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/POS/left_hand-pos_wrapper4.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/POS/left_hand-pos4.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/POS/right_hand-pos_wrapper2.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/POS/right_hand-pos2.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/POS/left_hand-pos_wrapper2.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/POS/left_hand-pos2.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/motorControl/left_leg-mc_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/motorControl/right_leg-mc_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/motorControl/left_leg-mc_remapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/motorControl/right_leg-mc_remapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/motorControl/right_arm-mc_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/motorControl/right_arm-mc_remapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/motorControl/left_arm-mc_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/motorControl/left_arm-mc_remapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/motorControl/torso-mc_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/motorControl/torso-mc_remapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/motorControl/head-mc_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/motorControl/head-mc_remapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/head-eb20-j0_1-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/head-eb21-j2_3-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/torso-eb5-j0_2-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/left_arm-eb2-j0_1-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/left_arm-eb4-j2_3-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/left_arm-eb31-j4_6-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/left_arm-eb23-j7_10-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/left_arm-eb25-j11_12-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/right_arm-eb1-j0_1-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/right_arm-eb3-j2_3-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/right_arm-eb30-j4_6-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/right_arm-eb22-j7_10-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/right_arm-eb24-j11_12-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/left_leg-eb8-j0_3-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/left_leg-eb9-j4_5-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/right_leg-eb6-j0_3-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/motorControl/right_leg-eb7-j4_5-mc.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/FT/left_arm-eb2-j0_1-strain.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/FT/right_arm-eb1-j0_1-strain.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/FT/left_leg-eb9-j4_5-strain.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/FT/right_leg-eb7-j4_5-strain.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/FT/left_leg-eb8-j0_3-strain.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/FT/right_leg-eb6-j0_3-strain.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/FT/left_leg-FT_remapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/FT/right_leg-FT_remapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/FT/left_arm-FT_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/FT/right_arm-FT_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/FT/left_leg-FT_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/FT/right_leg-FT_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/inertials/head-imuFilter_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/inertials/head-imuFilter.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/inertials/head-inertials_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/inertials/head-inertial.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/inertials/waist-inertial.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/inertials/waist-inertials_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/inertials/left_arm-eb4-j2_3-inertial.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/inertials/left_arm-inertials_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/inertials/right_arm-eb3-j2_3-inertial.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/inertials/right_arm-inertials_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/inertials/left_arm-eb2-j0_1-inertial.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/inertials/right_arm-eb1-j0_1-inertial.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/inertials/left_leg-eb9-j4_5-inertial.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/inertials/right_leg-eb7-j4_5-inertial.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/inertials/left_arm-imu_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/inertials/right_arm-imu_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/inertials/left_foot-imu_wrapper.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/inertials/right_foot-imu_wrapper.xml
NOT found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/battery/ergobattery_bat.xml
NOT found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/battery/ergocubbattery_bat.xml
NOT found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/wrappers/battery/ergobattery_bms.xml
NOT found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/hardware/battery/ergocubbattery_bms.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/calibrators/torso-calib.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/calibrators/left_leg-calib.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/calibrators/right_leg-calib.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/calibrators/left_arm-calib.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/calibrators/right_arm-calib.xml
found :/dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/calibrators/head-calib.xml

***************************************
FAIL ! not found 4 of 77 file included.
```

### Devices consistency check
```console
[DEBUG] Reading file /dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/./ergocub_all.xml
[DEBUG] yarprobotinterface: using xml parser for DTD v3.x
[DEBUG] Reading file /dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/./ergocub_all.xml
[INFO] Yarprobotinterface was started using the following enable_tags: 
[INFO] Yarprobotinterface was started using the following disable_tags: 
[ERROR] Failed to open file  file /dependencies/install/share/ICUBcontrib/robots/ergoCubSN000/./wrappers/battery/ergobattery_bat.xml included by  ergocub_all.xml at line 74
[FATAL] In file: ergobattery_bat.xml  included by:  ergocub_all.xml  at line:  74
Trace requested at /dependencies/yarp/src/libYARP_robotinterface/src/yarp/robotinterface/impl/XMLReaderFileV3.cpp:316 by code called from:
/dependencies/install/bin/../lib/libYARP_os.so.3(_Z16yarp_print_traceP8_IO_FILEPKcj+0x3b) [0x7f5b819af2fb]
/dependencies/install/bin/../lib/libYARP_robotinterface.so.3(+0x119ce) [0x7f5b81b219ce]
/dependencies/install/bin/../lib/libYARP_robotinterface.so.3(+0x4c18a) [0x7f5b81b5c18a]
/dependencies/install/bin/../lib/libYARP_robotinterface.so.3(+0x4aaef) [0x7f5b81b5aaef]
/dependencies/install/bin/../lib/libYARP_robotinterface.so.3(+0x54715) [0x7f5b81b64715]
/dependencies/install/bin/../lib/libYARP_robotinterface.so.3(+0x55b12) [0x7f5b81b65b12]
/dependencies/install/bin/../lib/libYARP_robotinterface.so.3(_ZN4yarp14robotinterface9XMLReader16getRobotFromFileERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEERKNS_2os10SearchableE+0x1939) [0x7f5b81b3f879]
yarprobotinterface(+0xb64b) [0x563d1969464b]
/dependencies/install/bin/../lib/libYARP_os.so.3(_ZN4yarp2os8RFModule9runModuleERNS0_14ResourceFinderE+0xf7) [0x7f5b81a0c667]
yarprobotinterface(+0x8dcf) [0x563d19691dcf]
/lib/x86_64-linux-gnu/libc.so.6(+0x29d90) [0x7f5b814c4d90]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0x80) [0x7f5b814c4e40]
yarprobotinterface(+0x8e95) [0x563d19691e95]
```

**Comment by pattacini** [comment]
Thanks for the quick fix @MSECode 👍🏻 
The CI is now ok ✅ 

I'll keep the PR in `changes requested` to address the following:

>After her review, @MSECode, can you please update the PR in order to already ship the same updates for ergoCubSN001?

**Comment by MSECode** [comment]
> Thanks for the quick fix @MSECode 👍🏻 The CI is now ok ✅
> 
> I'll keep the PR in `changes requested` to address the following:
> 
> > After her review, @MSECode, can you please update the PR in order to already ship the same updates for ergoCubSN001?

Oks,
I just commit the update for ergoCubSN001 as well

**Comment by valegagge** [comment]
We need also to update the [robots-configuration template](https://github.com/robotology/robots-configuration/tree/devel/iCubTemplates/iCubTemplateV6_0), but we'll address in another PR asap

**Pull request merged**
